src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/AspNetCoreFrontend.cs

```diff
@@ -64,7 +64,7 @@ string dmsPath
     {
         if (frontendResponse.LocationHeaderPath != null)
         {
-            string urlBeforeDmsPath = httpContext.Request.UrlWithPathSegment()[..^(dmsPath.Length + 1)];
+            string urlBeforeDmsPath = httpContext.Request.UrlWithPathSegment()[..^dmsPath.Length];
```

Comment:
bugfix

Reply:
This change causes a double slash in the location header between `data` and `ed-fi`. Pasted from postman: `http://localhost:5198/data//ed-fi/absenceEventCategoryDescriptors/c9437be7-8546-4237-8df0-069f8f504f50`

Reply:
That so weird. On my machine the "a" in "data" gets removed with the `+ 1`. I'll try something different.